packages/core/src/App/Containers/RealityCheckModal/brief-modal.jsx

```diff
@@ -1,96 +1,161 @@
-                                cancel_label={localize('Log out')}
-                                is_disabled={!values.interval || !isValid || isSubmitting}
-                                label={localize('Continue trading')}
-                                onCancel={logout}
-                            />
-                        </Modal.Footer>
-                    </Form>
-                )}
-            </Formik>
-        </Modal>
+    return (
+        <React.Fragment>
+            <DesktopWrapper>
+                <Modal
+                    className='reality-check'
+                    enableApp={enableApp}
+                    is_open={is_visible}
+                    disableApp={disableApp}
+                    has_close_icon={false}
+                    title={localize('Trading statistics report')}
+                    portalId='modal_root_absolute'
+                    width='720px'
+                >
+                    <Formik
+                        initialValues={{
+                            interval: '',
+                        }}
+                        validate={validateForm}
+                        onSubmit={onSubmit}
+                    >
+                        {({ errors, isSubmitting, isValid, values, touched, handleChange, handleBlur }) => (
+                            <Form noValidate>
+                                <Modal.Body>
+                                    <Message />
+                                    <IntervalField
+                                        values={values}
+                                        touched={touched}
+                                        errors={errors}
+                                        handleChange={handleChange}
+                                        handleBlur={handleBlur}
+                                    />
+                                </Modal.Body>
+                                <Modal.Footer has_separator>
+                                    <FormSubmitButton
+                                        className='reality-check__submit'
+                                        has_cancel
+                                        cancel_label={localize('Log out')}
+                                        is_disabled={!values.interval || !isValid || isSubmitting}
+                                        label={localize('Continue trading')}
+                                        onCancel={logout}
+                                    />
+                                </Modal.Footer>
+                            </Form>
+                        )}
+                    </Formik>
+                </Modal>
+            </DesktopWrapper>
+            <MobileWrapper>
+                <FadeWrapper is_visible={is_visible} className='reality-check' keyname='reality-check'>
+                    <div className='reality-check__wrapper'>
+                        <PageOverlay header={localize('Trading statistics report')} is_open={is_visible}>
```

Comment:
you can use MobileFullPageModal which is contained FadeWrapper,PageOverLay and Div100vhContainer

Reply:
Yea but I don't want to use `Div100vhContainer` here. Also I want to wrap inside another `div` here.